src/Helpers/Tokens/TokenVerifier.php

```diff
@@ -110,23 +110,19 @@ public function verify(string $token, array $options = []) : array
          * Audience checks
          */
 
-        $tokenAud = $verifiedToken->getClaim('aud', false);
-        if (! $tokenAud || (! is_string($tokenAud) && ! is_array($tokenAud))) {
+        $tokenAud = $verifiedToken->claims()->get('aud', false);
+        if (! is_array($tokenAud)) {
             throw new InvalidTokenException(
                 'Audience (aud) claim must be a string or array of strings present in the ID token'
             );
         }
 
-        if (is_array($tokenAud) && ! in_array($this->audience, $tokenAud)) {
+        if (! in_array($this->audience, $tokenAud)) {
```

Comment:
The new parser will convert 'aud' claim to an array so is_array check is not necessary anymore and elseif part too because $tokenAud can not be string
https://github.com/lcobucci/jwt/blob/058b61b376339208622b6dc9d38b70a122338c20/src/Token/Parser.php#L103